app/assets/v2/js/shared.js

```diff
@@ -220,13 +230,21 @@ var mutate_interest = function(bounty_pk, direction, data) {
 };
 
 
-var uninterested = function(bounty_pk, profileId) {
-  var request_url = '/actions/bounty/' + bounty_pk + '/interest/' + profileId + '/uninterested/';
+var uninterested = function(bounty_pk, profileId, slash = false) {
+  var slashed = '0';
+  var success_message = 'Contributor removed from bounty.';
+
+  if (slash) {
```

Comment:
I think it's better to send `true` and `false` instead of 0 and 1. And then [here](https://github.com/gitcoinco/web/pull/1343/files#diff-3e886cd5b50df8f0cabc86d33ca734e6R352) you can write something like `if slash == 'true'` ? cc: @mbeacom

Reply:
I agree with this.  Additionally, we could probably swap this out to simply use a POST param versus the URL defined variable, defaulting to `False` if the `request.POST` param isn't present.

Reply:
Updating the code to use a POST param, as the @mbeacom said.

Reply:
Do you think it's better send the POST params only when the `slash` is true?  Makes more sense to me always send it.